src/features/unified-share-modal/__tests__/UnifiedShareModal.stories.test.js

```diff
@@ -1,17 +1,25 @@
 describe('features/unified-share-modal/UnifiedShareModal', () => {
-    const USM_STORIES = [
+    const STORIES = [
         'features-unifiedsharemodal--basic',
         'features-unifiedsharemodal--with-shared-link',
         'features-unifiedsharemodal--with-autofocused-shared-link',
+        'features-unifiedsharemodal--with-form-only',
     ];
 
-    test.each(USM_STORIES)('looks visually correct when using story %s', async id => {
+    const MODAL_STORIES = [STORIES[0], STORIES[1], STORIES[2]];
+
+    test.each(STORIES)('looks visually correct when using story %s', async id => {
         const image = await BoxVisualTestUtils.takeScreenshot(id);
         return expect(image).toMatchImageSnapshot();
     });
 
-    test.each(USM_STORIES)('looks visually correct when button is clicked in story %s', async id => {
+    test.each(MODAL_STORIES)('looks visually correct when button is clicked in story %s', async id => {
```

Comment:
as a coding tip, `STORIES.slice(0,-1)` would do the same, and avoid keeping the arrays in sync. 

Here, tho, might actually be better to define each test with its own list of keys. later changes to the order of `STORIES` could lead to failing tests and confusion.